firefox_puppeteer/api/places.py

```diff
@@ -88,14 +88,22 @@ def restore_default_bookmarks(self):
 
               let observer = {
                 observe: function (aSubject, aTopic, aData) {
-                  Services.obs.removeObserver(observer, "bookmarks-restore-success");
+                  if ( aTopic === "bookmarks-restore-success") {
+                    Services.obs.removeObserver(observer, "bookmarks-restore-success");
```

Comment:
With this change we will not remove all registered observer topics. So what you wanna do is to call `removeObserver` for both registered topics at the beginning of this method.